website/views.py

```diff
@@ -42,6 +42,83 @@ def appointment():
 def ca_info():
     return render_template("ca_info.html")
 
+@main_blueprint.route("/assessment_form/<string:student_id>")
```

Comment:
I think it would be helpful to break down the submission route into individual pieces as currently it handles quite a lot of responsibilities,